### PR TITLE
UHack is Back!

### PR DESCRIPTION
**Pull request by gravelcycles**

### CODE REVIEW

**Comment by ndneighbor** [comment]
👍

File: src/all_events.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
 let all_events = [
   {
+    "name": "UHack is Back!",
+    "color": "#f37423",
+    "date": {
+      "from": 1512236561000,
+      "to": 1512322961000
+    },
+    "event_url": "http://uhackis.life",
```
Comment (jalvarado91) [inline src/all_events.js:9]: Url seems to be dead right now. Is this the url for the event?

**Review by jalvarado91 — COMMENTED** [review]
Question here

File: src/all_events.js (hunk 1 above)
Comment (gravelcycles) [reply src/all_events.js:9]: I think it's good now. Let me know

**Comment by jalvarado91** [comment]
Looks good now. Could you pull and rebase? New stuff got merged recently, and your branch would be behind.

**Pull request merged**